components/builder-depot/src/server.rs

```diff
@@ -1987,7 +2054,7 @@ where
         packages_pkg: get "/pkgs/:origin/:pkg" => list_packages,
         package_pkg_versions: get "/pkgs/:origin/:pkg/versions" => list_package_versions,
         package_pkg_latest: get "/pkgs/:origin/:pkg/latest" => show_package,
-        packages_version: get "/pkgs/:origin/:pkg/:version" => list_packages,
+        packages_for_version: get "/pkgs/:origin/:pkg/:version" => list_origin_packages_for_version,
```

Comment:
@chefsalim That's right -- channels are returned alongside actual package releases, so at the level above version, it doesn't quite fit (at least not yet). I like `_with_channels`, though; will change to use that.